packages/astro/vendor/vite/dist/node/chunks/dep-35df7f96.js

```diff
@@ -21319,7 +21319,9 @@ function buildHtmlPlugin(config) {
                                     .map((child) => child.content || '')
                                     .join('');
                                 // <script type="module">...</script>
-                                const filePath = id.replace(config.root, '');
+                                const filePath = id
+                                  .replace(config.root, '')
+                                  .replace(/^\/+/, '/') // Windows fix: replacing config.root sometimes leaves double leading slashes
```

Comment:
This needs to be a Vite PR. Internally, Vite is inconsistent with its Windows paths. For example, `id` isn’t normalized (could be backslashes or forward slashes). But `config.root` is normalized to forward slashes (`C:/Users/…`) on Windows. When it’s replaced, we get doubled slashes and other weirdness, so we need to do just a little cleanup here rather than rewriting all of Vite’s internals (this is the only place this URL-editing occurs)